ibm/resource_ibm_is_image_test.go

```diff
@@ -0,0 +1,92 @@
+package ibm
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.ibm.com/Bluemix/riaas-go-client/clients/compute"
+	"github.ibm.com/Bluemix/riaas-go-client/riaas/models"
+)
+
+func TestAccIBMISImage_basic(t *testing.T) {
+	var image *models.Image
+
+	href := fmt.Sprintf("cos://us-south/vnfimages/BIGIP-15.0.1-0.0.11.qcow2")
```

Comment:
why do we need href and operating declared we already added then as precheck to set the values via enviornmental variables